src/main/java/seedu/address/model/entity/body/BodyIdentificationNumber.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.entity.body;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+//@@author ambervoong
+/**
+ *  Represents a body's identification number in Mortago.
+ */
+public class BodyIdentificationNumber {
```

Comment:
Do we still need this class since `bodyIdentificationNumber` uses class `IdentificationNumber`?

Reply:
Thanks for the reminder! Moved to main/unused.